README.md

```diff
@@ -26,8 +26,13 @@ This is not an official Google product.
 
 ## Building Bloaty
 
-Bloaty uses CMake to build.  All dependencies are included as Git submodules.
-To build, simply run:
+Building Bloaty requires CMake and ``protoc``, the protobuf compiler. On Ubuntu, install them with:
+
+```
+$ sudo apt install cmake protobuf-compiler
+```
+
+All other dependencies are included as Git submodules. To build, run:
```

Comment:
We should probably mention that the build will use system-installed libprotobuf, re2, and capstone if they are available: https://github.com/google/bloaty/blob/master/CMakeLists.txt#L14-L16